pages/search/index.vue

```diff
@@ -103,11 +114,51 @@
 <script>
   import LoadingSpinner from '~/components/LoadingSpinner.vue';
   import axios from 'axios';
+  // TODO: cherry-pick Lodash methods? or load from CDN?
+  import _ from 'lodash';
```

Comment:
Maybe it's worth to investigate if we need lodash at all. [This](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore) is a handy overview.

Reply:
Where native equivalents exist, they have been switched to. Where not, Lodash's have been cherry-picked.